sonar-project.properties

```diff
@@ -0,0 +1,7 @@
+sonar.projectName=digital-marketplace
+sonar.sources=src/front-end,src/back-end
+sonar.sourceEncoding=UTF-8
+
+sonar.organization=bcgov-sonarcloud
```

Comment:
Yes this org corresponds to the installed dashboard: https://sonarcloud.io/organizations/bcgov-sonarcloud/projects